config/locales/server.en.yml

```diff
@@ -2819,6 +2819,18 @@ en:
       %{raw}
       ```
 
+    ignored_users_summary:
+      title: "Ignored User passed threshold"
+      subject_template: "New Ignored user passed threshold"
+      text_body_template: |
+        Hello,
+
+        This is an automated message from %{site_name} to inform you about a new ignored user who passed the number of ignores threshold which is: %{ignores_threshold}
```

Comment:
```suggestion
        This is an automated message from %{site_name} to inform you about a potentially problematic user who has been ignored by %{ignores_threshold} users. It would be smart to review their forum activity.
```